### PR TITLE
add a faster make-image.py

### PR DESCRIPTION
**Pull request by ArsenArsen**

### CODE REVIEW

File: scripts/init-sysroot.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+amd64_init() {
+	install -dm 750 "${sysroot}/home/managarm"
+	install -d "${sysroot}/boot/managarm"
+	install -d "${sysroot}/boot/grub"
+
+	# install kernel, prekernel
+	_umb="${sysroot}/usr/managarm/bin"
+	install_s -s "${_umb}"/eir-stivale "${sysroot}/boot/managarm"
+	install_s -s "${_umb}"/eir-mb1 "${sysroot}/boot/managarm"
```
Comment (Dennisbonke) [inline scripts/init-sysroot.sh:32]: We should really add `eir-mb2` here too. We build it but it never gets installed. While we're at it, we should update the grub entries to add multiboot 2 options too.

File: scripts/update-image.py [hunk 2]
```diff
@@ -1,599 +1,1031 @@
...
+                "-S",
+                "512",
+                "--offset",
+                str(start),
+                self.image,
+                str(size // 1024),
+            ]
+            logged_check_call(cmd)
+
+            # manupilate it using mtools
```
Comment (Dennisbonke) [inline scripts/update-image.py:821]: Typo, `manupilate` should be `manipulate`.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Looks ok to me, but I'll let @avdgrinten take a look too. One small nitpick tho.

File: scripts/.flake8 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 88
```
Comment (avdgrinten) [inline scripts/.flake8:2]: Why 88 and not 100?

Comment (ArsenArsen) [reply scripts/.flake8:2]: Matches `black`.

Comment (avdgrinten) [reply scripts/.flake8:2]: In xbstrap, we configure black to use a line length of 99, we should probably also do that here.

Comment (ArsenArsen) [reply scripts/.flake8:2]: ah, I didn't remember doing that, will change this evening

**Review by avdgrinten — APPROVED** [review]

**Pull request merged**